detox/src/environmentFactory.js

```diff
@@ -85,14 +85,6 @@ function _getFactoryClasses(deviceConfig) {
       runtimeDeviceFactoryClass = runtimeDeviceFactories.IosSimulator;
       break;
 
-    case 'ios.none':
-      envValidatorFactoryClass = envValidationFactories.Noop;
```

Comment:
Need to also delete the class itself https://github.com/wix/Detox/blob/dbe56701004bc5bc8ac8c490de113616ee1b1336/detox/src/validation/factories/index.js#L25

Reply:
Disagree! It is reused by ExternalValidator and by Android Emulator.